packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-settings.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import type { ConnectedProps } from 'react-redux';
+import {
+  Button,
+  Icon,
+  Toggle,
+  Label,
+  css,
+  spacing,
+} from '@mongodb-js/compass-components';
+import { toggleSettingsIsExpanded } from '../../modules/settings';
+import { toggleAutoPreview } from '../../modules/auto-preview';
+import { newPipelineFromText } from '../../modules/import-pipeline';
+import { collationCollapseToggled } from '../../modules/collation-collapser';
+import type { RootState } from '../../modules';
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  marginTop: spacing[2],
+});
+
+const buttonGroupStyles = css({
+  display: 'flex',
+});
+
+const extraSettingsGroupStyles = css({
+  display: 'flex',
+});
+
+const buttonStyles = css({
+  marginLeft: spacing[1],
+  marginRight: spacing[1],
+});
+
+const toggleStyles = css({
+  display: 'flex',
+  alignItems: 'flex-end',
+  marginLeft: spacing[1],
+  marginRight: spacing[1],
+});
+
+const toggleLabelStyles = css({
+  marginBottom: 0,
+  textTransform: 'uppercase',
+  marginLeft: spacing[1],
+});
+
+const PipelineSettings: React.FunctionComponent<PipelineSettingsProps> = ({
+  isCollationExpanded,
```

Comment:
Does this need to be part of the redux state?

Reply:
yeah to show the toggle icon on `collation` button.